Embedded System L1/AVR ATMEGA32/DIO/PROJECT/STD_TYPES.h

```diff
@@ -0,0 +1,21 @@
+/*********Author : Khalid Mosaad Zakaria */
+
+/*********Date   :   20/1/2021          */
+/*********Version: 1                     */
+/****************Desc: *******************/
+
+#ifndef	_STD_TYPES_H_
+#define _STD_TYPES_H_ 
+
+
+typedef unsigned char u8;
+typedef unsigned short int u16;
+typedef unsigned long int u32;
+typedef signed char s8;
+typedef signed short int s16;
+typedef signed long int s32;
+typedef float f32;
+typedef double f64;
+
+
+#endif
```

Comment:
-comments doesn't follow Sprints_CodeGuideLine of headers

![2](https://user-images.githubusercontent.com/44152298/127502084-967ab9b6-4fba-4be7-ba83-0af348f54c31.PNG)

[Sprints_CodeGuideLine.pdf](https://github.com/AhmedEssamMo/Sprints_Wave_9/files/6900596/Sprints_CodeGuideLine.pdf)